### ISSUE TITLE
Allow `assign` directive to chain directly off of the tagname interpolation

### ISSUE THREAD

**Issue by electrovir** [electrovir/element-vir#10]
Allow this:

```html
<${MyElement.assign({
    inputs: 'blah'
})}><${MyElement}>
```

instead of this:

```html
<${MyElement}
    ${assign({
        inputs: 'blah'
    })}
><${MyElement}>
```

**Comment by electrovir** [comment]
possible way to do this:

1. add `assign` method to `DelcarativeElementDefinition` which returns itself
2. when `assign` is called on `DelcarativeElementDefinition`, it assigns the input values to a new "assigned input" property on the created `DelcarativeElementDefinition`
3. `assign` will have to return a new instance of the `DelcarativeElementDefinition` so as to not modify the original instance.
4. The HTML template mapper will then check to see if this "assigned input" property is set, and if it is, it will extract it and modify the mapped template to include a value in the template that is a call to the `assign` directive..

**Comment by electrovir** [comment]
released in v14.1.0
